core/include/kdl_codyco/sensors.hpp

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (C) 2015 Fondazione Istituto Italiano di Tecnologia
+ * Author: Silvio Traversaro
+ * email:  silvio.traversaro@iit.it
+ * Permission is granted to copy, distribute, and/or modify this program
+ * under the terms of the GNU General Public License, version 2 or any
+ * later version published by the Free Software Foundation.
+ *
+ * A copy of the license can be found at
+ * http://www.robotcub.org/icub/license/gpl.txt
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details
+ */
+
+#ifndef IDYNTREE_CORE_SENSORS_HPP
+#define IDYNTREE_CORE_SENSORS_HPP
+
+namespace KDL {
+    class Wrench;
+}
+
+
+#include <string>
+
+namespace KDL {
+namespace CoDyCo {
+
+    enum SensorType
+    {
+        SIX_AXIS_FORCE_TORQUE = 0,
+        NR_OF_SENSOR_TYPES = 1
```

Comment:
Just a personal taste: I don't like to define the possible number of enum values in the enum itself.